accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -900,6 +913,8 @@ if ($a === $b) {
     $foo = $bar ?? $a ?? $b;
 } elseif ($a > $b) {
     $foo = $a + $b * $c;
+} elseif ($a <=> $c) {
+    $foo += $bar % $a & $b ** $c;
```

Comment:
This looks like meaningless code(especially `elseif ($a <=> $c)` doesn't make sense), what is it intended for?

Reply:
i guess is just an example, to show how to use the `<=>` operator

Reply:
This exactly has no sense.
But this exactly shows the syntax of less thought operators.

That was the idea behind that. :)